assets/src/dashboard/app/views/shared/storyListView.js

```diff
@@ -316,9 +347,14 @@ export default function StoryListView({
                     />
                   ) : (
                     <>
-                      <SelectableParagraph>
+                      <Text
+                        tabIndex={0}
```

Comment:
I think this should be `<Heading size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.XXXSMALL} as="h4">` according to figma.

h4 is a bit arbitrary, but this is my thinking: 
![Screen Shot 2021-02-25 at 2 15 52 PM](https://user-images.githubusercontent.com/10720454/109218736-080e5200-7774-11eb-86b5-7fee2339dd90.png)

So when we've got the other parts of the dashboard merged for header and filters above the view, that state that shows the view, that could be  h3, and then it'd make a lot of sense for these titles to be nested in the heading outline of that.

Reply:
Looks like there's no `XXX_SMALL` for headline, so I set it to `XX_SMALL` which is the necessary `14px`